controllers/notifications.js

```diff
@@ -0,0 +1,56 @@
+const notifications=require('../models/notification');
+
+// GET / -Returns all notifications from the db
+exports.returnNotifications=async(req,res)=>{
+    const userId = res.locals.user._id;
+    const notifications=await notifications.find({toUser:userId}).populate('eventId');
+    res.status(200).send({
+        success:true,
```

Comment:
Just send notifications, no need to make object. Refer to tags or forums routes.